src/Sylius/Bundle/AdminBundle/Resources/config/services/form.xml

```diff
@@ -273,5 +273,11 @@
         <service id="sylius_admin.form.type.zone" class="Sylius\Bundle\AdminBundle\Form\Type\ZoneType" >
             <tag name="form.type" />
         </service>
+
+        <service id="sylius.attribute_type.select" class="Sylius\Component\Attribute\AttributeType\SelectAttributeType">
+            <tag name="sylius.attribute.type" attribute-type="select" label="Select"
+                 form-type="Sylius\Bundle\AttributeBundle\Form\Type\AttributeType\SelectAttributeType"
+                 configuration-form-type="Sylius\Bundle\AdminBundle\Form\Type\AttributeType\Configuration\SelectAttributeConfigurationType" />
+        </service>
```

Comment:
As this service is not a form type, I would be for moving it into another XML file